packages/samples/headless-react/src/index.tsx

```diff
@@ -4,10 +4,16 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
+const strictMode = true;
```

Comment:
I could remove those changes, I only added them to be easily able to toggle strict mode.